meilisearch-http/src/option.rs

```diff
@@ -49,15 +65,21 @@ pub struct Opt {
     pub no_analytics: bool,
 
     /// The maximum size, in bytes, of the main lmdb database directory
-    #[structopt(long, env = "MEILI_MAX_MDB_SIZE", default_value = "107374182400")] // 100GB
+    #[structopt(long, env = "MEILI_MAX_MDB_SIZE", parse(try_from_str=parse_size), default_value = "107374182400")]
+    // 100GB
     pub max_mdb_size: usize,
 
     /// The maximum size, in bytes, of the update lmdb database directory
-    #[structopt(long, env = "MEILI_MAX_UDB_SIZE", default_value = "107374182400")] // 100GB
+    #[structopt(long, env = "MEILI_MAX_UDB_SIZE", parse(try_from_str=parse_size), default_value = "107374182400")]
+    // 100GB
     pub max_udb_size: usize,
 
     /// The maximum size, in bytes, of accepted JSON payloads
-    #[structopt(long, env = "MEILI_HTTP_PAYLOAD_SIZE_LIMIT", default_value = "104857600")] // 100MB
+    #[structopt(
+        long,
+        env = "MEILI_HTTP_PAYLOAD_SIZE_LIMIT",
+        default_value = "104857600"
+    )] // 100MB
```

Comment:
```suggestion
    #[structopt(
        long,
        env = "MEILI_HTTP_PAYLOAD_SIZE_LIMIT",
        parse(try_from_str=parse_size),
        default_value = "104857600"
    )] // 100MB
```

You don't use your method for the payload size?